SpatialGDK/Documentation/content/get-started/example-project/exampleproject-cloud-deployment.md

```diff
@@ -192,7 +192,11 @@ You can find out more in the SpatialOS Worker SDK documentation: [launch confgur
   This is: `C:/...<filepath>.../UnrealGDKExampleProject/spatial/snapshots/default.snapshot`. </br></br>
   * **Region**: The real-world geographical location that you want your cloud deployment hosted in. </br>You can change this by selecting a different region from the drop-down list. You might prefer the region you are in.</br></br>
 1. You can also choose to add simulated players via the dialogue box or skip this and move on to item _4._</br></br>
-  To do this, you launch a simulated players deployment with your your game deployment via the **Simulated Players** section of the Cloud Deployment dialog box:</br></br>
+  Build out the simulated player clients (which will run on Linux in the cloud) using the following command:</br></br>
+  ```
+  BuildWorker.bat <YourProject>SimulatedPlayer Linux Development <YourProject>.uproject 
```

Comment:
Similarly here, we should point out both the engine plugin and project plugin alternatives.